packages/flutter/test/material/dropdown_test.dart

```diff
@@ -3165,4 +3166,77 @@ void main() {
         ..rrect(rrect: const RRect.fromLTRBXY(0.0, 0.0, 112.0, 47.0, 2.0, 2.0), color: Colors.grey[50], hasMaskFilter: false)
     );
   });
+
+  group('feedback', () {
+    late FeedbackTester feedback;
+
+    setUp(() {
+      feedback = FeedbackTester();
+    });
+
+    tearDown(() {
+      feedback.dispose();
+    });
+
+    Widget feedbackBoilerplate({bool? enableFeedback}) {
+      return MaterialApp(
+        home : Material(
+          child: DropdownButton<dynamic>(
+            iconSize: 24,
+            elevation: 16,
+            enableFeedback: enableFeedback,
+            underline: Container(
+              height: 2,
+              color: Colors.deepPurpleAccent,
+            ),
+            onChanged: onChanged,
+            items: <String>['One', 'Two']
+                .map<DropdownMenuItem<String>>((String value) {
+              return DropdownMenuItem<String>(
+                value: value,
+                child: Text(value),
+              );
+            }).toList(),
+          ),
+        ),
+      );
+    }
+
+    testWidgets('Dropdown with enabled feedback', (WidgetTester tester) async {
+      const bool enableFeedback = true;
+
+      await tester.pumpWidget(feedbackBoilerplate(enableFeedback: enableFeedback));
+
+      await tester.tap(find.byType(DropdownButton));
+      await tester.pump(const Duration(seconds: 1));
+      await tester.tap(find.byType(InkWell).first);
```

Comment:
If the goal here is to tap the dropdown button, then tester.tap(find.text('One')) might be clearer. Otherwise we need an explanation of why we're looking for an InkWell, and why it's the first InkWell...

Reply:
Same problem I can't use `tester.tap(find.text('One'))` to tap on the DropdownButton because the `InkWell` containing the text isn't visible yet.

Reply:
If feedbackBoilerplate() specifies `value: 'One'` then `tester.tap(find.text('One'))` will tap the dropdown button.

Reply:
Thanks worked like a charm. :smile: